.gitignore

```diff
@@ -11,4 +11,5 @@ atlassian-ide-plugin.xml
 *.iml
 
 Google Docs Share/src/main/amp/config/alfresco/web-extension/custom-slingshot-googledocs-context.xml
-Google Docs Repository/src/main/oauth-client-resources/**
\ No newline at end of file
+Google Docs Repository/src/main/oauth-client-resources/**
+Google Docs Repository/src/main/amp/config/alfresco/subsystems/googledocs/drive/google-customResponse-context.xml
```

Comment:
I notice that we also have:

https://github.com/Alfresco/googledrive/tree/develop/Google%20Docs%20Repository/src/main/customResponseContextFiles

Related ?